src/App.vue

```diff
@@ -23,19 +24,29 @@
 import { defineComponent } from 'vue';
 import 'vue-loaders/dist/vue-loaders.css';
 import SearchResults from "./components/SearchResults.vue";
+import DarkModeToggle from './components/DarkModeToggle.vue';
 
 export default defineComponent({
   name: "app",
   components: {
+    DarkModeToggle,
     SearchResults
-  },
+},
   created() {
     document.title = 'SilverStripe GitHub Issues';
   }
 });
 </script>
 
 <style>
+  html.dark {
+    color-scheme: dark;
+  }
+
+  .dark {
+    color: #31a8ff;
```

Comment:
I assume this is where the text is getting its blue from. Can we please swap this out for a nice light grey? My brain is wired to think "blue means link" and now suddenly everything on the page looks like a link to me.

Reply:
Yep, that's the main "text" colour - will opt for something light gray and with high-enough contrast

Reply:
Updated screenshot and PR 👍